css/events.css

```diff
@@ -6,6 +6,10 @@ Events Styles
 ===================================
 */
 
+header {
+	min-height: auto;
+}
+
```

Comment:
Can you adjust or add some MQ styles to improve responsive styles?
At smaller viewport widths, the padding for **.event-detail-container** and **cta-container** are not adjusting, making the content/imgs viewable:

768px all sections should have some adjusted padding

![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/014612dc-2898-49a0-af20-d003ecedb043)


470px needs some adjustments as well
![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/e1af59c3-bc2c-42e6-89f4-cf569278b477)
![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/a78c42bf-3b8b-4f43-8f0c-7bb3611621cd)


please let me know if you need more clarification - thank you!

Reply:
I made some adjustments, please take a look